examples/streaming/README.md

```diff
@@ -0,0 +1,164 @@
+
+## Snap Plugin Go Library: Stream Collector Plugin Example
+Here you will find an example plugin that covers the basics for writing a stream collector plugin.
+
+## Plugin Naming, Files, and Directory
```

Comment:
This is consistent with the other example plugins. You might have a point about changing it on all of them. What document do you think it should be pointed at?